core/tests/protractor_desktop/classroomPage.js

```diff
@@ -31,41 +31,44 @@ describe('Classroom page functionality', function() {
   var classroomPage = null;
   var libraryPage = null;
 
-  beforeAll(function() {
+  beforeAll(async(done) => {
```

Comment:
This is a weird syntax and doesn't follow any of the examples [here](https://github.com/oppia/oppia/wiki/Migrating-End-to-End-Tests-to-Async-Await)... what is it doing?

Similarly for the various wrappings of async() below...

Reply:
Oh -- is this ES6? We might want to be consistent throughout the e2e tests and do this a standard way in all files...

Reply:
Hi Sean, I plan to grep all the async function and convert them to asyn(). The function keyword is no longer required in the latest ES standard. Wdyt?

Reply:
I'm fine with anything as long as we're consistent. Make sure to update the wiki page to match!

Reply:
Yep. Will do, thanks!

Reply:
Actually, I'm going to revert these back to "function async(" instead. This is because converting the 900+ occurences of "function async" to the new syntax with arrow functions will introduce changes in the scope of the variables used inside the function and that can cause unnecessary failures in the e2e test. Thanks!

Reply:
Done.

Reply:
I've confirmed that this is fixed, so I'm resolving this comment